polymerfire.html

```diff
@@ -10,4 +10,7 @@
 <link rel="import" href="firebase-document.html">
 <link rel="import" href="firebase-query.html">
 <link rel="import" href="firebase-messaging.html">
-<link rel="import" href="firebase-storage-script.html">
```

Comment:
@tjmonsi Why you deleted `firebase-storage-script.html`? :-/ Is it no longer needed?